lib/libzpool/Makefile.am

```diff
@@ -12,6 +12,8 @@ AM_CFLAGS += $(NO_UNUSED_BUT_SET_VARIABLE)
 # Includes kernel code generate warnings for large stack frames
 AM_CFLAGS += $(FRAME_LARGER_THAN)
 
+AM_CFLAGS += -DLIB_ZPOOL_BUILD
```

Comment:
Can you explain why `LIB_ZPOOL_BUILD` is needed?

Reply:
libzpool is just the zfs mod in userspace.  For the `zfs_mod_supported_feature` test, the kernel and libzpool builds are the same in that they know about all the features/properties they are building.  libzfs on the other hand needs to check with sysfs to see what the kernel knows about.